Makefile

```diff
@@ -30,8 +30,9 @@ test-ios: generate-xcodeproj
 		| $(XCPRETTY)
 
 test-playgrounds: test-macos
+	# this isn't right...
 	find . \
-		-path '*.playground/*' \
+		-path '*.playground/Pages/*.xcplaygroundpage/*' \
```

Comment:
this wasn't working with multiple playground pages, and it still isn't working. couldn't figure out the magic.